### ISSUE TITLE
Empty lines in method body

### ISSUE THREAD

**Issue by blablatdinov** [wemake-services/wemake-python-styleguide#2486]
# Rule request

Empty lines in the method body are bad. I propose prohibit their use or limit it count in method body

## Thesis

I received a PR with the same code

```python
    def to_representation(self, data):
        result = None

        self.get_person(data)

        if self.person and self.person.use_other_veh:

            result = super().to_representation(data)

        return result
```

## Reasoning

I think, if method have few empty lines, it not holistic, and it makes sense to divide the method into several. Yegor Bugaenko's [post](https://www.yegor256.com/2014/11/03/empty-line-code-smell.html) about empty lines

**Comment by sobolevn** [comment]
Interesting, I think we can say that there must a maximum of empty lines in a function's body.
For example: a function with 2 lines can only have 0 empty lines.
But, a functions with 4 lines, can have 1 empty line.

I think we need to come up with a ratio 🤔

**Comment by blablatdinov** [comment]
@sobolevn Thank you for answer! I agree with you. Maybe made this functional configurable (like `max-arguments`)

**Comment by blablatdinov** [comment]
@sobolevn Yes, of course. But I don't have experience in develop plugins for `flake8`, and I will be glad of any support.

At the end of the month, I will make a report linter at the "X5 Group" internal meetup (greetings from Artem Kalinin). I would like to add a slide about the extension of this tool to my presentation

**Comment by sobolevn** [comment]
Awesome! 

So, basically you need to take a look at `visit_FuncDef` or `visit_any_function` methods.
Inside you will see the full `ast` of a function defition. 

Next, collect all lines that have 0 nodes on it to find empty ones. And collect the final number of lines.

Finally, decide should we raise a violation or not.
Any help I can provide - I will :)

**Comment by blablatdinov** [comment]
@sobolevn I have increment: create class for checking empty lines in functions or method body. In the next commit I planning add parameter for configuration this rule. Please check my PR

**Comment by blablatdinov** [comment]
@sobolevn I'm push changes that make this rule configurable, please check it
